content/en/boilerplates/revision-tags-preamble.md

```diff
@@ -0,0 +1,11 @@
+---
+---
+Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone.
+[Revision tags](/blog/2021/revision-tags/) are a solution to this. [Revision tags](/blog/2021/revision-tags/) are
```

Comment:
nit: the blog might be out of date at some point, we should link to our official docs and improve them? maybe we can do this at a later point though.

Reply:
Let's do this as part of our follow-up work to clean up the documentation.

Reply:
This is also changed/changing in https://github.com/istio/istio.io/pull/10086.

```suggestion
[Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-revision-tag) are a solution to this. Revision tags are
```